src/angular/src/app/pages/settings/path-pairs.component.ts

```
@@ -0,0 +1,154 @@
+  // Add-new form state
+  adding = false;
+  addForm: Omit<PathPair, 'id'> = this.emptyForm();
+
+  // Double-click delete confirmation
+  confirmingDeleteId: string | null = null;
+  private confirmResetTimer: ReturnType<typeof setTimeout> | null = null;
+  private subscriptions: Subscription[] = [];
+
+  ngOnDestroy(): void {
+    this.clearConfirmTimer();
+    this.subscriptions.forEach((s) => s.unsubscribe());
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)